backend/src/main/java/com/trackpack/app/model/trackingmodel/ShipmentTracking.java

```diff
@@ -0,0 +1,40 @@
+package com.trackpack.app.model.trackingmodel;
+
+import com.trackpack.app.model.usermodel.User;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+
+@Data
+@AllArgsConstructor
+public class ShipmentTracking {
+
+    private String id;
+    private String href;
+    private String carrier;
+    private String trackingCode;
+    private String carrierTrackingUrl;
+    private LocalDateTime trackingDate;
+    private String status;
+    private LocalDateTime statusChangeDate;
+    private String statusChangeReason;
+    private int weight;
+    private LocalDateTime estimatedDeliveryDate;
+    private Address addressFrom;
+    private Address addressTo;
+    private ArrayList<CheckPoint> checkpoint;
+    private User user;
```

Comment:
Maybe we need to store only user's ID?

Reply:
Need to discuss it. From where we get this id, should we have it on shipment, what kind of relation between user and shipment tracking.